server/app/models.py

```diff
@@ -237,12 +237,12 @@ def get_backups(self, assignment, num_backups=10):
         return self.backups(group, assignment).fetch(num_backups)
 
     def submissions(self, group, assignment):
-        """A query that fetches all backups for a group and assignment."""
+        """A query that fetches all submissions for a group and assignment."""
         members = self.members(group)
         return Submission.query(
             Submission.submitter.IN(members),
             Submission.assignment == assignment
-        ).order(-Backup.server_time)
+        ).order(-Submission.server_time)
```

Comment:
That looked wrong ...  `-Backup.server_time` might be a relic of the days when submissions were backups.